courses/ai-for-finance/practice/intro_tf_data_keras_sequential.ipynb

```diff
@@ -0,0 +1,937 @@
+{
```

Comment:
<p>Last sentence - replace "of" with "we", ...do you see any issues with the examples we have to train on?  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>